x-pack/plugins/monitoring/server/lib/alerts/fetch_cpu_usage_node_stats.ts

```diff
@@ -147,38 +342,52 @@ export async function fetchCpuUsageNodeStats(
     },
   };
 
-  try {
-    if (filterQuery) {
-      const filterQueryObject = JSON.parse(filterQuery);
-      params.body.query.bool.filter.push(filterQueryObject);
-    }
-  } catch (e) {
-    // meh
+  if (filterQuery) {
```

Comment:
**nit**: the `filterQuery` param is not JSON parsed in a try-catch anymore [here](https://github.com/elastic/kibana/pull/159351/files?diff=split&w=0#diff-6e02cdce4e86b6d1e470f7af6628eb78c967ecf90a0c2e106accadcb8a69be1bR66) (even if before it was not handling anyway the cought error), I'm just wondering if there is any chance the filterQuery param will come in as an unparseable string.

Reply:
Ah I forgot about coming back to this, let me add in the try/catch again and just throw an actual error this time!